sematic/ui/src/components/RunNode.tsx

```diff
@@ -11,7 +11,17 @@ import { Handle, NodeProps, Position } from "react-flow-renderer";
 import { Run } from "../Models";
 import RunStateChip from "./RunStateChip";
 
-function getColor(futureState: string, theme: Theme): PaletteColor {
+function getColor(
+    futureState: string, isCloned: boolean, theme: Theme
+): PaletteColor {
+  if (isCloned) {
+    return {
+       light: "#155020",
+       main: "#103515",
+       dark: "#052010",
+       contrastText: "#fff",
```

Comment:
I tried to see what it would look like to render the cached Runs using a different palette.

The tough color choice aside, it now seems a little overloaded though, so now I'm leaning towards reverting this until we have a more thought-through plan to represent information as a whole in the rendered boxes.

Reply:
Switched to the already-used grey / disabled palette instead of overloading the existing color scheme.

Reply:
Reverted color change to implement a comment.